### PR TITLE
feat: adjust schedule

### PR DESCRIPTION
**Pull request by HassanBahati**
- add workshops

### CODE REVIEW

**Review by Amanya-Brian — APPROVED** [review]
Thank you for the update

**Comment by basiima** [comment]
image

Seen this repetition on day 2 and day 3

File: components/schedule/data.js [hunk 1]
```diff
@@ -154,29 +178,44 @@ export const PYCON_DAY3_SCHEDULE = [
     speaker: "",
   },
   {
-    title: "Empowering Data Scientists: Unleashing the Potential of Azure Cloud Services",
+    title:
+      "Empowering Data Scientists: Unleashing the Potential of Azure Cloud Services",
     duration: "11:00 - 11:30",
     speaker: "Lorna Aine",
+    workshopTitle:
+      "Python for Robust Data Infrastructure: AirQo#&39;s Air Quality Monitoring in Africa",
```
Comment (HassanBahati) [inline components/schedule/data.js:186]:
```suggestion
      "Python for Robust Data Infrastructure: AirQo's Air Quality Monitoring in Africa",
```

File: components/schedule/data.js [hunk 2]
```diff
@@ -154,29 +178,44 @@ export const PYCON_DAY3_SCHEDULE = [
...
+    workshopSpeaker: "Noble Mutabazi, Richard Sserunjogi, Noah Nsimbe",
   },
   {
     title: "Cultivating Excellence: Nurturing Growth as a Web Developer",
     duration: "11:30 - 12:00",
     speaker: "Abudi Mutamba",
+    workshopTitle:
+      "Python for Robust Data Infrastructure: AirQo#&39;s Air Quality Monitoring in Africa",
```
Comment (HassanBahati) [inline components/schedule/data.js:194]:
```suggestion
      "Python for Robust Data Infrastructure: AirQo's Air Quality Monitoring in Africa",
```

File: components/schedule/data.js [hunk 3]
```diff
@@ -154,29 +178,44 @@ export const PYCON_DAY3_SCHEDULE = [
...
+    workshopSpeaker: "Noble Mutabazi, Richard Sserunjogi, Noah Nsimbe",
   },
   {
-    title: "Enhancing Python Application Scalability: Infrastructure, CI/CD, Testing",
+    title: "The Art of Defensive programming",
     duration: "12:30 - 13:00",
-    speaker: "Abigail Afi Gbadago",
+    speaker: "Joylynn Kirui",
+    workshopTitle:
+      "Python for Robust Data Infrastructure: AirQo#&39;s Air Quality Monitoring in Africa",
```
Comment (HassanBahati) [inline components/schedule/data.js:210]:
```suggestion
      "Python for Robust Data Infrastructure: AirQo's Air Quality Monitoring in Africa",
```

**Comment by HassanBahati** [comment]
> image
> 
> Seen this repetition on day 2 and day 3

is it supposed to be on only one?

**Comment by basiima** [comment]
I am not sure, it just seemed off to me

**Comment by Amanya-Brian** [comment]
Is it for breakout sessions?

**Comment by HassanBahati** [comment]
> image
> 
> Seen this repetition on day 2 and day 3

what exactly do you mean by repetition... do you mean that the workshop `Python for Robust Data Infrastructure: AirQo's Air Quality Monitoring in Africa` appears on both day 2 and day 3 ?

**Comment by HassanBahati** [comment]
> Is it for breakout sessions?

something along those lines...workshops running concurrently with other talks...

**Comment by HassanBahati** [comment]
> I am not sure, it just seemed off to me

i think the representation there is for a workshop spanning more than one period.....

kindly advice on a better way to present such a scenario...

**Comment by basiima** [comment]
Alright if that session will span across other different sessions then that's fine. we can merge this

**Comment by Amanya-Brian** [comment]
Cool

**Pull request merged**